rpmbuild.c

```diff
@@ -659,6 +674,10 @@ int main(int argc, char *argv[])
     case 's':
 	ba->buildAmount |= RPMBUILD_PACKAGESOURCE;
 	break;
+    case 'd':
```

Comment:
This looks fishy. The `'r'` case also gets 
```
ba->buildAmount |= RPMBUILD_PREP;
```
and I fail to see why this should not be needed here. Basically the only difference here should be to not fall through to the `'s'` case.